extensions/amp-story-audio-sticker/amp-story-audio-sticker.md

```diff
@@ -0,0 +1,253 @@
+---
+$category@: presentation
+formats:
+  - websites
+teaser:
+  text: Fill this in with teaser text to improve SEO. Use the component description.
+---
+
+<!--
+  All documentation starts with frontmatter. Front matter organizes documentation on amp.dev
+  and improves SEO.
+  * Include the relevant category(ies): ads-analytics, dynamic-content, layout, media, presentation, social, personalization
+  * List applicable format(s): websites, ads, stories, email
+  * Do not include markdown formatting in the frontmatter - plain text and punctionation only!
+  * Remove this comment!
+-->
+
+# amp-story-audio-sticker
+
+<!--
+  If the component is relevant for more than one format and operates differently between these
+  formats, include and filter multiple content blocks and code samples.
+-->
+
+## Usage
+
+One to three paragraphs explaining the component usage. List important functionality. Explain why developers care about it.
+
+[filter formats=“websites”]
+
+Below is an example for websites.
+
+[example preview="inline" playground="true" imports="amp-story-audio-sticker"]
+
+```html
+<amp-story-audio-sticker required-attribute>
+  I am a hello world inline executable code sample for websites!
+</amp-story-audio-sticker>
```

Comment:
Remove any boilerplate text that could show in the docs, and fix the `required-attribute` in the code

Reply:
Remove almost everything here. This would be updated after all the implementation is ready.